pkg/restapi/issuer/operation/operations.go

```diff
@@ -30,8 +30,11 @@ const (
 	login    = "/login"
 	callback = "/callback"
 	retrieve = "/retrieve"
+	revokeVC = "/revokeVC"
```

Comment:
it should be either revoke or retrieveVC; probably revoke

Reply:
done